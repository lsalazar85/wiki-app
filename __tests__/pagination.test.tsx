/**
 * @jest-environment jsdom
 */

import { render  } from '@testing-library/react'
import Pagination from "../components/Pagination";

describe('Pagination', () => {
    it('should render successfully', () => {
        const { asFragment } = render(
            <Pagination
                pageCount={34}
                setPage={() => jest.fn()}
                currentPage={2}
            />
        )

        expect(asFragment()).toMatchSnapshot();
        expect(asFragment()).toBeDefined()
    })
})