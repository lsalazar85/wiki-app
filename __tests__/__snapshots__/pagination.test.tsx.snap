// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should render successfully 1`] = `
<DocumentFragment>
  <div
    class="styles__PaginationContainer-sc-1pxi0jf-0 hyggEk"
  >
    <div
      class="styles__PaginationDiv-sc-1pxi0jf-1 dSpzUZ"
    >
      <button
        class="styles__ButtonContainer-sc-9v32al-0 kRkStR"
      >
        Prev
      </button>
      <span
        class="styles__PaginationText-sc-1pxi0jf-2 eSGdAL"
      >
        2 of 34
      </span>
      <button
        class="styles__ButtonContainer-sc-9v32al-0 kRkStR"
      >
        Next
      </button>
    </div>
  </div>
</DocumentFragment>
`;
